tf_encrypted/convert/register.py

```diff
@@ -441,6 +471,8 @@ def nodef_to_public_pond(converter, x):
             nums = x.attr["value"].tensor.float_val
         elif dtype == tf.float64:
             nums = x.attr["value"].tensor.float_val
+        elif dtype == tf.int32:
```

Comment:
should `int32` be supported above as well?

Reply:
not sure, I understand. did you mean `int64`?